File: 1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array/1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array.py
class Solution(object):
    def minNumberOperations(self, target):
        """
        :type target: List[int]
        :rtype: int
        """
        changes = target[0]
        for i in range(1, len(target)):
            changes += max(0, target[i]-target[i-1])
        return changes

File: 1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array/1526-minimum-number-of-increments-on-subarrays-to-form-a-target-array.cpp
class Solution {
public:
    int minNumberOperations(vector<int>& target) 
    {
        int changes = target[0];
        
        for(int i = 1; i < target.size();i++)
        {
            if(target[i] - target[i-1] > 0)
                changes += target[i] - target[i-1];
        }
        
        return changes;
    }
};